app.js

```diff
@@ -1,5 +1,11 @@
+import Config from './config/config';
+import { User, Product} from './models/models';
+
 class App {
```

Comment:
app should not be a class, we don't create an instance of it when we run it

Reply:
Removed class implementation for app